test/utils.py

```diff
@@ -556,7 +556,7 @@ def reset_hba(self):
         """Remove any HBA rules that were added after the last call to commit_hba"""
         with self.hba_path.open() as f:
             hba_contents = f.read()
-        committed = hba_contents[hba_contents.find("# committed-rules\n"):]
+        committed = hba_contents[hba_contents.find("# committed-rules\n") :]
```

Comment:
This is weird. Why would it do that? Do we know?

Reply:
Putting whitespace around the slice operator is recommended by pep8 for complex expressions: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices

Apparently black considers this function call complex enough to trigger adding a single space. 

I don't feel strongly either way. And it's not possible to configure this behaviour differently. Based on your comment it seems like you don't feel strong about it either, mostly finding the change unexpected. So I'll merge the PR.